Dockerfile-client

```diff
@@ -17,5 +18,10 @@ RUN yarn run build
 RUN echo '*/1 * * * * cd /airnode && yarn run dev:invoke' > local-cron
 RUN /usr/bin/crontab /airnode/local-cron 
 
+HEALTHCHECK --interval=5m --timeout=2m \
+    CMD (strace -p $(pgrep crond) -s9999 -o cronLog.txt) & sleep 70 ; kill $! \
+        && cat cronLog.txt | grep crond > /dev/null; if [ 0 != $? ]; then kill 1; fi && rm cronLog.txt
```

Comment:
This HEALTHCHECK passes if the cron command has executed atleast once in the following 70 seconds. If the cron command is not found to be executed then the container exits .If we don't want the container to exit we can replace kill 1 with sleep 70 which will cause the HEALTHCHECK to exceed the timeout and change the status to `unhealthy` .

Reply:
I think -s9000 should be -s 9000 based on https://man7.org/linux/man-pages/man1/strace.1.html (still trying to test locally)

Reply:
it should work even without the space, in fact -p should also work without the space ut I added it for clarity, You can have a look [here](https://itectec.com/unixlinux/bash-how-to-view-the-output-of-a-running-process-in-another-bash-session/)